framework/src/executioners/Steady.C

```diff
@@ -48,6 +48,7 @@ Steady::init()
   }
 
   checkIntegrity();
+  _problem.execute(EXEC_PRE_INITIAL);
```

Comment:
This will absolutely bypass initialSetup() for each system that attempts to execute here.